### DISCUSSION TITLE
Sourcing portal page is getting stuck when user logs in through Chrome and edge browsers

### DISCUSSION THREAD

**Discussion by Aparna-karri591** [Sunbird-coKreat/Community#86] [Bugs]
**Steps to replicate:**

1. Open sourcing portal in chrome browser
2. Login as a sourcing admin
3. Not able to click any tab, Page unresponsive popup is coming
4. And If we clear the cache it is working. Note: For few users, even after clearing the cache, it does not work. 

![image](https://github.com/Sunbird-coKreat/Community/assets/106593060/9c845dff-2d83-4edc-8058-ffe9706f817f)

Bug: https://project-sunbird.atlassian.net/browse/CO-421

following are the details which I am using
**Chrome Version**: 113.0.5672.127
**OS:** Windows 10 Pro
**RAM**:16GB
**Note:** For few users, it did not work in Chrome, edge. The only browser that worked was Opera

**Please Add your comments here If anyone facing the same issue**

@manimegalai-smohan @vinukumar-vs

**Comment by mmanimegalai** [comment]
@Aparna-karri591 I have encountered this issue at least 3 times, in the last may be 2-3 months. I faced the issue in chrome and clearing cache or opening in incognito mode had zero effect. In fact Kamesh also had similar issue in Chrome, edge sometime back. The only browser that worked was opera. cc @kameshbhr

**Comment by mmanimegalai** [comment]
The outcome of the analysis is updated in https://project-sunbird.atlassian.net/browse/CO-421
This will be prioritized and fixed in the forthcoming release
